### PR TITLE
PMM-6141

### PR DESCRIPTION
**Pull request by IvanaWzientkova**
[PMM-6141](https://jira.percona.com/browse/PMM-6141)

### CODE REVIEW

File: pmm-app/tests/verifyPMMDemo_test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature('Verify PMMDemo side');
+
+Scenario('PMM-T363 - Verify Copyrights & Legal section elements', async (I, pmmDemoPage) => {
+  I.amOnPage(pmmDemoPage.url);
+  I.wait(5);
```
Comment (puneet0191) [inline pmm-app/tests/verifyPMMDemo_test.js:5]: Why?

Comment (IvanaWzientkova) [reply pmm-app/tests/verifyPMMDemo_test.js]: Already changed! :)

File: pmm-app/tests/verifyPMMDemo_test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Scenario('PMM-T363 - Verify Copyrights & Legal section elements', async (I, pmmDemoPage) => {
+  I.amOnPage(pmmDemoPage.url);
+  I.waitForVisible(pmmDemoPage.fields.title, 30);
+  pmmDemoPage.verifyCopyrightsAndLegal();
+  I.amOnPage(pmmDemoPage.url + pmmDemoPage.mongoDBDashbordUrl);
+  pmmDemoPage.verifyCopyrightsAndLegal();
+});
+
+Scenario(
+  'PMM-T364 - Verify PMM settings returns Access denied error',
```
Comment (puneet0191) [inline pmm-app/tests/verifyPMMDemo_test.js:12]: @IvanaWzientkova could you please add @not-pr-pipeline tag to each scenario, we don't these tests running for Grafana Dashboard PR's, we will have another job to execute them, also use tag @pmm-demo, so we can group scenarios for pmm-demo together.

File: pmm-app/tests/verifyPMMDemo_test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  }
+);
+
+Scenario(
+  'PMM-T364 - Verify PMM settings returns Access denied error @not-pr-pipeline @pmm-demo',
+  async (I, pmmDemoPage, pmmInventoryPage) => {
+    I.amOnPage(pmmDemoPage.url + pmmInventoryPage.url);
+    I.waitForVisible(pmmDemoPage.fields.accessDenied, 30);
+  }
+);
```
Comment (atymchuk) [inline pmm-app/tests/verifyPMMDemo_test.js:28]:
```suggestion
```

Comment (atymchuk) [reply pmm-app/tests/verifyPMMDemo_test.js:28]: I must be a duplicate of the scenario right above

Comment (IvanaWzientkova) [reply pmm-app/tests/verifyPMMDemo_test.js:28]: Sorry, I forgot to change test case number. It's not duplicate that's two similar tests. Will push change right now. :)

**Pull request merged**